cross-project-tests/debuginfo-tests/dexter/feature_tests/subtools/test/err_syntax_mline.cpp

```diff
@@ -4,14 +4,11 @@
 //      Check directives are in check.txt to prevent dexter reading any embedded
 //      commands.
 //
-// Note: Despite using 'lldb' as the debugger, lldb is not actually required
-//       as the test should finish before lldb would be invoked.
-//
```

Comment:
Is the comment not still true?

Reply:
It's true that the debugger isn't invoked, but since I replaced the reference to `lldb` with `%dexter_regression_test_debugger_args` (since either is just as correct, and the latter prevents any misunderstanding of the test depending on LLDB), the comment no longer makes sense (since we are not necessarily using 'lldb' as the debugger), and I don't think we need a comment to specify that the debugger will not be run.